src/Microsoft.AspNetCore.Server.Kestrel/Internal/Infrastructure/MemoryPoolIteratorExtensions.cs

```diff
@@ -279,7 +270,7 @@ public static bool GetKnownMethod(this MemoryPoolIterator begin, out string know
 
             if ((value & _mask4Chars) == _httpGetMethodLong)
             {
-                knownMethod = HttpGetMethod;
+                knownMethod = HttpMethods.Get;
```

Comment:
Might be a tiny performance degradation since the value is changing from `const` to `static readonly`.  Is there a reason the properties on `HttpMethods` are `static readonly` rather than `const`?  I see the `HttpMethods` type is new for 1.1, so we should be able to change the properties to `const` now.  After 1.1 ships it would be a breaking change.

I don't see any possible way the value of these properties would change in the future, so `const` should be safe.